docker/scripts/test-container-image.sh

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env bash
 . "$(realpath $(dirname $0))/common.sh"
 
-ROOT="$(get_root_dir)"
-
 CACHE_DIR=$(obtain_cache_dir "$0")
-DEPLOYMENT_DIR="${CACHE_DIR}/deployment"
-DEPLOYMENT_CONFIG_FILE="${ROOT}/scripts/deployment/ethereum-local.json"
+
+ARTIFACTS_DIR="${CACHE_DIR}/deployments/artifacts/local"
```

Comment:
Nitpick: here we're using `ARTIFACTS_DIR`, while in other scripts we use `ARTIFACT_DIR`.
I think we should use `ARTIFACTS_DIR` also in other scripts, and then we would be consistent everywhere (also with commands).